### DISCUSSION TITLE
Agent Committee Meeting, Today, Wednesday 1/11/2022

### DISCUSSION THREAD

**Discussion by lin-fred** [ArctosDB/arctos#6140] [Agent Committee]
Hey everyone,

Reminder the next Agent meeting is, today, the 11th at 1:30pm MTN time. You can see the [Arctos calendar](https://calendar.google.com/calendar/u/2?cid=amt2OWY4a2UwYXBxamJzOG52c3FybTRyODRAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ) for details and zoom link. Please contact me if you have issues using the zoom information.

Take a look at the agents project page https://github.com/orgs/ArctosDB/projects/16 under the Agent Committee Discussion/Verbatim Agents. Look over the issues, we will be going over them in order from top to bottom. Please add any that you want to discuss!

Also, we will spend the first 30 minutes to review the current Documentation with Agents, editing the documentation and updating it.

Agenda is here:

https://docs.google.com/document/d/1QC63PF_qj6W4b2-f7BgP9L6MTvR8fpgICEL2b6bWpW8/edit#

Looking forward to seeing everyone!

Lindsey

**Comment by droberts49** [comment]
I'll be out of the office next week so won't be at our meeting on 2/8.
Dawn

On Wed, Jan 11, 2023 at 9:50 AM Lindsey NMMNHS ***@***.***>
wrote:

> Hey everyone,
>
> Reminder the next Agent meeting is, today, the 11th at 1:30pm MTN time.
> You can see the Arctos calendar
> <https://calendar.google.com/calendar/u/2?cid=amt2OWY4a2UwYXBxamJzOG52c3FybTRyODRAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ>
> for details and zoom link. Please contact me if you have issues using the
> zoom information.
>
> Take a look at the agents project page
> https://github.com/orgs/ArctosDB/projects/16 under the Agent Committee
> Discussion/Verbatim Agents. Look over the issues, we will be going over
> them in order from top to bottom. Please add any that you want to discuss!
>
> Also, we will spend the first 30 minutes to review the current
> Documentation with Agents, editing the documentation and updating it.
>
> Agenda is here:
>
>
> https://docs.google.com/document/d/1QC63PF_qj6W4b2-f7BgP9L6MTvR8fpgICEL2b6bWpW8/edit#
>
> Looking forward to seeing everyone!
>
> Lindsey
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/ArctosDB/teams/agents-committee/discussions/16>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AD6P4NFYNW6JLXJTU2DXQGDWR3JELANCNFSM6AAAAAATYGEQVM>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
*Dawn Roberts** |* *Senior Director of Collections*
*Pronouns: she/her/hers*
The Chicago Academy of Sciences / Peggy Notebaert Nature Museum
2430 North Cannon Drive, Chicago, IL 60614 | naturemuseum.org
<http://www.naturemuseum.org/>

Collections Facility and Office
4001 N Ravenswood Avenue, suite 201, Chicago, IL 60613 | 773-755-5125
*Wonder is in Our Nature™*
